content/courses/advanced-ada/parts/data_types/shared_variable_control.rst

```diff
@@ -484,17 +484,641 @@ these array declarations are equivalent:
         - **Briefly** discuss :ada:`Full_Access_Only`
         - Add to previous section!
 
-..
-    TO BE DONE:
 
-    Package System.Atomic_Operations
-    --------------------------------
+.. _Adv_Ada_Package_System_Atomic_Operations:
+
+Atomic operations
+-----------------
+
+.. note::
+
+    This feature was introduced in Ada 2022.
+
+Ada offers four child packages to handle atomic operations. Those packages are
```

Comment:
It would be simpler to just say "Ada offers four packages to", since the detail that they are child packages is about to be mentioned next.

Reply:
Yes, I totally agree with you.